public/src/utils/KeyManager.js

```diff
@@ -0,0 +1,34 @@
+export class KeyManager {
+  constructor(p5) {
+    this.keysPressed = [];
+    this.commands = [
+      {
+        name: "screenshot",
+        keys: [17, 88], // ctrl + x
+        execute: p5.save.bind(p5),
```

Comment:
this `bind` usage here is the same deal as in the [`renderShape` function](https://github.com/glitch-dot-cool/paint-with-friends/blob/616ac39b8addb4726597c8d2af7d91f0e51d8ff9/public/src/utils/drawing.js#L30). `bind` is a little confusing, but [this description from MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind) summarizes it better than I would be able to: 
> The bind() method creates a new function that, when called, has its [`this`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this) keyword set to the provided value

basically, it's just making sure that the function has access to the `p5` object.